crowbar_framework/lib/crowbar/product.rb

```diff
@@ -29,7 +29,7 @@ def self.is_ses?
 
     def self.ses_platform
       # Returns the platform used for SES
-      "suse-12.2"
+      "suse-12.3"
```

Comment:
I could revert this to "suse-12.2" and we can create separate card for SES deployment investigation, what do you think?